src/panels/lovelace/common/compute-card-element.js

```diff
@@ -0,0 +1,21 @@
+const CARD_TYPES = [
```

Comment:
Let's not put this method in common. It's only useful for the Lovelace components (`hui-`). So let's keep it in the panel folder. Maybe add a common folder there ?

Reply:
that's what I did ^^

Reply:
doh 🤦‍♂️